xc/device_common.ml

```diff
@@ -367,20 +367,20 @@ let qmp_send_cmd_internal connection domid cmd =
     | Qmp.Success(None, _)
     | Qmp.Event(_) as resp ->
       let resp' = Qmp.string_of_message resp in
-      debug "%s: skipping unexpected QMP response from domid %d: %s"
-        __LOC__ domid resp';
+      debug "%s: skipping QMP message while waiting for %s: %s"
+        __LOC__ id resp';
```

Comment:
Maybe we should show the `id` in the debug message above `QMP command with id %d for domid %d...`

Reply:
`debug "QMP command for domid %d: %s" domid msg';` contains the `id` because it is part of the command.

Reply:
Oh, I see! Good :)